frontend/src/Editor/Components/components.js

```diff
@@ -529,6 +529,7 @@ export const componentTypes = [
     },
     styles: {
       textColor: { type: 'color', displayName: 'Text Color' },
+      radioButtonColor: { type: 'color', displayName: 'Radio Button Color' },
```

Comment:
`activeColor` might be a better name for this property?

Reply:
for displayName too ?

Reply:
yes